src/inet/node/leach/LeachNode.ned

```diff
@@ -0,0 +1,19 @@
+//
+// SPDX-License-Identifier: LGPL-3.0-or-later
+//
+
+package inet.node.leach;
+
+import inet.node.inet.AdhocHost;
+import inet.routing.leach.Leach;
+
+module LeachNode extends AdhocHost {
```

Comment:
The standard LEACH protocol doesn't specify the protocol for the Physical and MAC layers, however for my implementation I've used 802.15.4 standard that covers those layers as I was simulating a WSN as realistically as possible. Hence the reference to the Texas Instruments CC2520 in the .INI file comments as well.

So I thought it would be best to explicitly showcase which parameters have been set in the simulation .INI file so whoever is using it can customize as needed.

Reply:
okay. I get this.